rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/TaskanaRestExceptionHandler.java

```diff
@@ -222,7 +222,7 @@
           // value for the error output and want to use the contains performance boost of a HashSet.
           List<String> enumConstants =
               Arrays.stream(targetType.getEnumConstants())
-                  .map(Object::toString)
+                  .map(t -> ((Enum<?>) t).name())
```

Comment:
Our coding guidelines state, that using method references is the preferred way. Please replace this lamdas statement with 2 method references. Thanks!